dashboard/src/components/UpgradeForm/UpgradeForm.tsx

```diff
@@ -7,28 +7,20 @@ import PackageHeader from "components/PackageHeader/PackageHeader";
 import PackageVersionSelector from "components/PackageHeader/PackageVersionSelector";
 import { push } from "connected-react-router";
 import * as jsonpatch from "fast-json-patch";
-import {
-  AvailablePackageDetail,
-  InstalledPackageDetail,
-} from "gen/kubeappsapis/core/packages/v1alpha1/packages";
 import * as yaml from "js-yaml";
 import { useEffect, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { Action } from "redux";
 import { ThunkDispatch } from "redux-thunk";
 import { deleteValue, setValue } from "../../shared/schema";
-import { IPackageState, IStoreState } from "../../shared/types";
+import { IStoreState } from "../../shared/types";
 import * as url from "../../shared/url";
 import DeploymentFormBody from "../DeploymentFormBody/DeploymentFormBody";
 import LoadingWrapper from "../LoadingWrapper/LoadingWrapper";
 import "./UpgradeForm.css";
 
 export interface IUpgradeFormProps {
-  installedAppAvailablePackageDetail: AvailablePackageDetail;
-  installedAppInstalledPackageDetail: InstalledPackageDetail;
-  selectedPackage: IPackageState["selected"];
-  chartsIsFetching: boolean;
-  error?: Error;
```

Comment:
So these props stopped being used or moved into state in a previous commit?

Reply:
Yeah, I forgot to remove them when I moved them to the state. Now I'm just passing the version for the fixed upgrade.